#Publish joint state
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

  #Position controller
joint1_position_controller:
  type: effort_controllers/JointPositionController
  joint:  Base_Shoulder
  pid: {p: 100.0,i: 0,d: 0}

joint2_position_controller:
  type: effort_controllers/JointPositionController
  joint: Shoulder_Elbow
  pid: {p: 100.0,i: 0,d: 0}

joint3_position_controller:
  type: effort_controllers/JointPositionController
  joint: Elbow_Wrist_1
  pid: {p: 100.0,i: 0,d: 0}

joint4_position_controller:
  type: effort_controllers/JointPositionController
  joint: Wrist_1_Wrist_2
  pid: {p: 100.0,i: 0,d: 0}

joint5_position_controller:
  type: effort_controllers/JointPositionController
  joint: Wrist_2_Wrist_3
  pid: {p: 100.0,i: 0,d: 0}


joint6_position_controller:
  type: effort_controllers/JointPositionController
  joint: Wrist_3_Tool_Flange
  pid: {p: 100.0,i: 0,d: 0}